### ISSUE TITLE
Suggestion: Scale Images and Video 

### ISSUE THREAD

**Issue by greg200** [jashwant/kickass-slider#10]
It would be absolutely amazing if we could scale the dimensions of images and videos within the slider. 

As I need to always resize my images before adding to the slider, using Iresize. 

Great Plugin though!

**Comment by jashwant** [comment]
I agree. But I am the only developer behind this. I am freelancer and developing something opensource costs me alot. Hope for the best. Wait till some really 'good' developers notice this plugin and contribute in it. I'll try to add this feature in my next release though. However, I would say re-sizing images before adding to slider, is a good idea. It reduces image size and thus page loads faster.

**Comment by greg200** [comment]
I've managed to Scale the images by adjusting the load-styles in wp-admin. I was just wondering if you could potentially make a max-height for the uploaded images within Kickass.

**Comment by jashwant** [comment]
I'll try to include your suggestions in next version. Until then if you add something new or fix something. Please share it by pushing it.

**Comment by greg200** [comment]
Is it possible to add a max-height of the images to 728px somehow?

**Comment by jashwant** [comment]
You can add a new css rule.
